tests/v2/validation/charts/monitoring.go

```diff
@@ -181,59 +182,51 @@ func checkPrometheusTargets(client *rancher.Client) (bool, error) {
 	return statusInit, nil
 }
 
-// editAlertReceiver is a private helper function
+// editAlertReceiverRoute is a private helper function
 // that edits alert config structure to be used by the webhook receiver.
-func editAlertReceiver(alertConfigByte []byte, origin string, originURL *url.URL) ([]byte, error) {
-	alertConfig, err := config.Load(string(alertConfigByte))
-	if err != nil {
-		return nil, errors.Wrapf(err, "failed to unmarshal alert config")
-	}
-
-	alertConfig.Global = &config.GlobalConfig{
-		ResolveTimeout: alertConfig.Global.ResolveTimeout,
+func editAlertReceiverRoute(origin string, originURL *url.URL) ([]byte, error) {
+	config := alertconfig.Config{}
+	resolveTimout, _ := model.ParseDuration("5m")
+
+	config.Global = &alertconfig.GlobalConfig{
+		SlackAPIURL:    "https://api.slack.com",
+		ResolveTimeout: resolveTimout,
+		SMTPRequireTLS: false,
```

Comment:
This could be omitted, I think, but it doesn't matter either way. I copied the same way to create an alert config from here. https://github.com/rancher/rancher/blob/b98b9915f8bd8ac714087c24b69be3ce0f873847/pkg/controllers/managementuserlegacy/alert/manager/alertmanager.go#L127